src/inmanta/agent/agent.py

```diff
@@ -754,52 +713,49 @@ async def dryrun(self, dry_run_id: uuid.UUID, version: int) -> Apireturn:
         return 200
 
     async def do_run_dryrun(self, version: int, dry_run_id: uuid.UUID) -> None:
-        async with self.dryrunlock:
-            async with self.ratelimiter:
```

Comment:
Why do we remove the ratelimiter here?

Reply:
the queue does that now

Reply:
How?

Reply:
We know that we can run only one task at a time, because only a single logical thread drains the queue,...